src/radio/radio-group.component.ts

```diff
@@ -88,6 +93,10 @@ export class RadioGroup implements AfterContentInit, AfterViewInit, ControlValue
 	 */
 	static radioGroupCount = 0;
 
+	@Input() orientation: "horizontal" | "vertical" = "horizontal";
+
+	@Input() placement: "left" | "right" =  "left";
```

Comment:
We have `placement` here and `labelLeft` in radio.component ... we should standardize on one ... 

Maybe `labelPlacement: "left" | "right"`? `placement` is fairly generic and I'm not sure if it would be confusing here